code-challenges.test.js

```diff
@@ -23,16 +33,41 @@ const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "s
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// create function named colors with a parameter of array
+const colors = (array) => {
+    // use the shift method with the argument to remove the first item in the array
+    array.shift()
+    // return array with the sort and reverse methods, sorting in reverse alphabetical order.
+    return array.sort().reverse()
+}
+
+// I'm sure there's a way to actually randomize it but I couldn't quite figure out how to properly use the sort function. I know it has defaults for basic values like numbers and letters but I couldn't understand how to customize things to do what I would've wanted it to do. Just kept getting confused.
+
```

Comment:
Jackson, thank you. Thank you for being honest AND doing your best.  Here's a great resource that discusses the sort method for shuffling. [click here](https://www.freecodecamp.org/news/how-to-shuffle-an-array-of-items-using-javascript-or-typescript/). You have the right idea, just were uncertain of how to manipulate the method.  Keep going!